app/ledger.js

```diff
@@ -867,7 +867,11 @@ var run = (delayTime) => {
   }
 
   if (delayTime === 0) {
-    delayTime = client.timeUntilReconcile()
+    try {
+      delayTime = client.timeUntilReconcile()
+    } catch (ex) {
+      delayTime = random.randomInt({ min: 1, max: 10 * msecs.minute })
```

Comment:
if this random delay happens to be small enough, is it still possible for `run` to be invoked before wallet initialization? i'm not understanding how this change fixes the problem stated in the PR